src/pages/Attendance.jsx

```diff
@@ -13,43 +13,138 @@ import ChurchIcon from "@mui/icons-material/Church"
 import ArrowLeftIcon from "@mui/icons-material/ArrowLeft"
 import ArrowRightIcon from "@mui/icons-material/ArrowRight"
 import SettingsIcon from "@mui/icons-material/Settings"
+import AddBoxOutlinedIcon from "@mui/icons-material/AddBoxOutlined"
+import IndeterminateCheckBoxOutlinedIcon from "@mui/icons-material/IndeterminateCheckBoxOutlined"
 
 import Layout from "../components/Layout/Layout"
 import Title from "../components/shared/Title"
 import { mcyMember } from "../data/mcyMember"
 
 const Attendance = () => {
-  const [value, setValue] = React.useState(dayjs())
   const [isOpenCalendar, setIsOpenCalendar] = React.useState(false)
   const [selectedLeader, setSelectedLeader] = React.useState(null)
-  const [isChecked, setIsChecked] = React.useState(false)
+  const [adultCount, setAdultCount] = React.useState(0)
+  const [memberCount, setMemberCount] = React.useState(0)
+  const totalCount = adultCount + memberCount
+  const [attendanceAllData, setAttendanceAllData] = React.useState({})
+  const [isChecked, setIsChecked] = React.useState({})
+  const currentDayOfWeek = dayjs().day()
+  const daysToLastSunday = currentDayOfWeek === 0 ? 7 : currentDayOfWeek
+  const lastSunday = dayjs().subtract(daysToLastSunday, "day")
+  const [value, setValue] = React.useState(dayjs(lastSunday))
```

Comment:
```suggestion
  const [adultCount, setAdultCount] = React.useState(0)
  const [memberCount, setMemberCount] = React.useState(0)
  const totalCount = adultCount + memberCount
  const [attendanceAllData, setAttendanceAllData] = React.useState({})
  const [isChecked, setIsChecked] = React.useState({})
  const currentDayOfWeek = dayjs().day()
  const daysToLastSunday = currentDayOfWeek === 0 ? 7 : currentDayOfWeek
  const lastSunday = dayjs().subtract(daysToLastSunday, "day")
  const [value, setValue] = React.useState(dayjs(lastSunday))
```
여기 너무 지저분해서 종류별로 개행으로 구분. useState끼리 모아두고 관련 변수 끼리 모아두는 식으로

그리고 왠만하면 React.useEffect React.useState로 사용하지 말고 그냥 useState, useEffect로 사용해도댐

Reply:
@ykss useState 관련은 나중에 브랜치 새로 파서 한 번에 전체 파일에 적용할게!